src/core_plugins/kibana/public/management/app.html

```diff
@@ -18,10 +18,10 @@
         </span>
         <a
           ng-if="sectionName"
-          ng-repeat="item in section.items.inOrder"
+          ng-repeat="item in section.visibleItems"
           class="kuiLocalTab"
-          ng-class="{ 'kuiLocalTab-isSelected': item.active, 'kuiLocalTab-disabled': !item.url }"
-          kbn-href="{{::item.url}}"
+          ng-class="{ 'kuiLocalTab-isSelected': item.active, 'kuiLocalTab-disabled': item.disabled || !item.url }"
+          kbn-href="{{::item.disabled ? '' : item.url}}"
```

Comment:
Yeah. There's a missing class to address the style issues. I think the problem is we just never had a use case for a disabled tab in the localNav until now, so it went unnoticed.